scripts/ts-copy.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-env node */
+// @flow
+
+const fs = require('fs-extra');
+const globby = require('globby');
+const path = require('path');
+
+(async () => {
```

Comment:
as we use this for build-time only, I think it's safe to use sync apis - I'd do either that, or add a catch to the async function to make sure we see errors if any

Reply:
It still throws errors

```
(node:35662) UnhandledPromiseRejectionWarning ... specific error message
```

`flow-copy-src` is using the same pattern